templates/taxi/car_confirmation_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <p>Are you sure you want to delete {{ car.name }}?</p>
+  <form action="" method="post">
+  {% csrf_token %}
+  <input type="submit" value="Yes, delete" class="btn btn-danger">

```

Comment:
it would be good to add cancel button

Reply:
![image](https://user-images.githubusercontent.com/115401096/227733731-2e2c88c2-768d-4e8a-96cc-efc269802cb1.png)
Done!